techmatters/terraso-mobile-client
# feat: Add images to slope shape info sheet

## Description
As suggested by the [Figma](https://www.figma.com/file/KMJbhjVyrrLFHj6jGcX9lN/LandPKS-Design-Prototype-working-file?type=design&node-id=5257-12606&mode=design&t=DMo9UTasep7Q2iMj-0) design for the Slope Shape info sheet, add the six images with their titles and labels. 

They are currently PNGs, there is a future issue to decide if they should be SVGs: #1114 

### Checklist
- [ ] English strings reviewed and copyedited --> I'm gonna assume this is part of the PR and verification process unless someone tells me otherwise
- [x] Images are compressed (TinyPNG or svgo)

### Related Issues
Second PR to finish work that implements #853 

### Verification steps
1. Navigate to a site's Slope Shape screen
2. Press the info button
3. Scroll down to see the added content
